src/contrib/mlir/pass/mlir_subgraph_extraction.cpp

```diff
@@ -284,6 +291,20 @@ bool MLIRSubgraphExtractionPass::is_supported_mlir_op(std::shared_ptr<Node> node
             return false;
     }
 
+    if (TI(ngraph::op::Maximum) == TI(*node) || TI(ngraph::op::Minimum) == TI(*node))
+    {
+        // TODO: Remove this when MLIR has float point cmp support
+        if (!node->input(0).get_element_type().is_integral())
+            return false;
+    }
+
+    if (TI(ngraph::op::Greater) == TI(*node) || TI(ngraph::op::Less) == TI(*node))
+    {
+        // TODO: Remove this when MLIR has float point cmp support
+        if (!node->input(0).get_element_type().is_integral())
+            return false;
+    }
```

Comment:
if op is Greater or Less and the prev condition is not met, shouldn't we return true here since we aren't evaluating Greater/Less properties anywhere below? Same for the rest of ops?